app/src/main/java/ai/elimu/appstore/synchronization/AppListActivity.java

```diff
@@ -24,9 +25,9 @@
 
     private List<Application> mApplications;
 
-    private ArrayAdapter mAdapterApps;
+    private AppListAdapter mAdapterApps;
```

Comment:
Try to keep the name of the property similar to the type. If the type is `AppListAdapter`, but the property name is `adapterApp`, it becomes harder to read the code below.